xml/Microsoft.Extensions.Compliance.Classification/DataClassification.xml

```diff
@@ -55,9 +55,11 @@
         <Parameter Name="value" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="taxonomyName">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="taxonomyName">The name of the data taxonomy this classification belongs to.</param>
+        <param name="value">The taxonomy-specific value representing the classification.</param>
+        <summary>
+            Initializes a new instance of the <see cref="T:Microsoft.Extensions.Compliance.Classification.DataClassification" /> struct.
+            </summary>
```

Comment:
I can't help but notice that all closing tags are misindented...

Reply:
https://dev.azure.com/ceapex/Engineering/_workitems/edit/930482